### PR TITLE
sentence-by-sentence pipeline

### PR DESCRIPTION
**Pull request by mssammon**
option to process TextAnnotations sentence-by-sentence. The goal is to allow more robust handling of documents -- catch exceptions for individual sentences, but continue processing afterwards. This may need some additional changes to specific handlers to throw appropriate exceptions, but this PR provides the basic pipeline behavior and an appropriate test.

### CODE REVIEW

**Comment by danyaljj** [comment]
@mssammon are the files checked in?

**Comment by mssammon** [comment]
I didn't push the right branch... now they're there

File: core-utilities/README.md [hunk 1]
```diff
@@ -377,6 +377,7 @@ argument to the `Annotator` constructor. This is used for lazy
 initialization, if active, in which case the first call to `getView()`
 will call `initialize()` with this configuration. 
 
+One key configuration default, AnnotatorConfig.
```
Comment (danyaljj) [inline core-utilities/README.md]: Is this sentence complete?

**Comment by bhargav** [comment]
Jenkins on morgoth is failing because the Gurobi 6.5 license expired. Unfortunately, I cannot seem to renew it or get a license for 6.5 anymore. Only new licenses are available for Gurobi 7+ version.

File: core-utilities/README.md (hunk 1 above)
Comment (mssammon) [inline core-utilities/README.md]: good catch -- thanks.

**Pull request merged**